packages/contractkit/src/wrappers/MultiSig.ts

```diff
@@ -42,7 +42,8 @@ export class MultiSigWrapper extends BaseWrapper<MultiSig> {
       if (
         transaction.data === data &&
         transaction.destination === destination &&
-        transaction.value === '0'
+        transaction.value === '0' &&
```

Comment:
Can you remind me why we check that `transaction.value == '0'` as opposed to checking that it matches txObject?

Reply:
because we currently only submit transactions with 0 value (line 55).  this method could be generalized even more by having it take in a `value` parameter.  

although this feature wasn't needed yet. 

and I'm not sure if it's even needed at all since the `data` field of the transaction its approving/executing will have the any `value` it needs for that transaction.  my understanding is this `value` is if there will be a transfer from multisig to the other contract while transacting

Reply:
done